src/Microsoft.OData.Extensions.Client.Abstractions/Microsoft.Extensions.OData.V3Client.Abstractions.Nightly.nuspec

```diff
@@ -18,8 +18,8 @@
         </dependencies>
     </metadata>
   <files>
-    <file src="$ProductRoot$\Desktop\Microsoft.Extensions.OData.V3Client.Abstractions.dll" target="lib\netstandard2.0" />
-    <file src="$ProductRoot$\Desktop\Microsoft.Extensions.OData.V3Client.Abstractions.pdb" target="lib\netstandard2.0" />
-    <file src="$ProductRoot$\Desktop\Microsoft.Extensions.OData.V3Client.Abstractions.xml" target="lib\netstandard2.0" />
+    <file src="$ProductRoot$\Desktop\Microsoft.OData.Extensions.V3Client.Abstractions.dll" target="lib\netstandard2.0" />
```

Comment:
@xuzhg, should "Desktop" be removed?

Reply:
We should. Please file a PR to remove, But, be patient, there are many places need to change.